metadata.json

```diff
@@ -77,6 +77,10 @@
       "name": "puppet",
       "version_requirement": ">= 7.0.0 < 9.0.0"
     },
+    {
```

Comment:
please don't add the module here. The `requirements` key is not used for modules. `dependencies` is used for modules (see above).